src/node/history.h

```diff
@@ -383,6 +381,35 @@ namespace ccf
     std::optional<ResultCallbackHandler> on_result;
     std::optional<ResponseCallbackHandler> on_response;
 
+    void execute_pending(kv::Version v)
```

Comment:
So that flushes already executed entries in the pending_inserts queue into the history unless I am missing something. execute_pending() doesn't seem like the best name for it.?